apps/trading/pages/_document.page.tsx

```diff
@@ -15,7 +15,7 @@ export default function Document() {
           rel="preload"
           href="/AlphaLyrae.woff2"
```

Comment:
This needs changing to just .woff as well

Reply:
updated